website/docs/mutation.md

```diff
@@ -3,7 +3,7 @@ id: mutation
 title: Mutation
 ---
 
-#Gatekeeper mutation
+## Gatekeeper mutation
```

Comment:
We can probably remove this line completely since website automatically adds title as header

Reply:
Thank you for your review! I deleted it.